server/src/test/java/org/apache/iotdb/db/integration/IoTDBDeleteTimeseriesIT.java

```diff
@@ -86,8 +86,8 @@ public void testDeleteTimeseries() throws Exception {
       }
       statement.execute("DELETE timeseries root.turbine1.d1.s1");
       statement.execute(
-          "create timeseries root.turbine1.d1.s1 with datatype=DOUBLE, encoding=PLAIN, compression=SNAPPY");
-      statement.execute("INSERT INTO root.turbine1.d1(timestamp,s1) VALUES(2,1.1)");
+          "create timeseries root.turbine1.d1.s1 with datatype=INT64, encoding=PLAIN, compression=SNAPPY");
+      statement.execute("INSERT INTO root.turbine1.d1(timestamp,s1) VALUES(2,5)");
```

Comment:
This test aims to test deleting and creating a different data type timeseries with the same name. Please do not change it : )

Reply:
fixed.